sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/ChangeFeedOffset.scala

```diff
@@ -34,7 +34,7 @@ private[cosmos] case class ChangeFeedOffset
   override def json(): String = jsonPersisted
 }
 
-private[spark] object ChangeFeedOffset {
+private[cosmos] object ChangeFeedOffset {
```

Comment:
Is this change intentional?
How was it working before?

Reply:
It is intentional - before only classes form the com.azure.cosmos.spark package (or "sub packages") were able to access this internal api. I know extended visibility to all packages under com.azure.cosmos because internal PAI access from one of the Bridge classes is needed.